src/components/SearchfoxLink.svelte

```diff
@@ -0,0 +1,69 @@
+<script>
+  const MozillaCentralURL = "https://searchfox.org/mozilla-central/";
+  const MozillaMobileURL = "https://searchfox.org/mozilla-mobile/";
+
+  export let appName;
+  export let metricName;
+
+  const SearchfoxMap = {
+    burnham: "", // (?)
+
+    fenix: "fenix",
+
+    "fenix-nightly": "fenix",
+
+    "firefox-android-beta": "fenix",
+
+    "firefox-android-nightly": "fenix",
+
+    "firefox-android-release": "fenix",
+
+    "firefox-desktop": "", // (?)
+
+    "firefox-focus-ios": "focus-ios",
+
+    "firefox-for-echo-show": "firefox-echo-show",
+
+    "firefox-for-fire-tv": "firefox-tv",
+
+    "firefox-ios-beta": "firefox-ios",
+
+    "firefox-ios-dev": "firefox-ios",
+
+    "firefox-ios-release": "firefox-ios",
+
+    "firefox-klar-ios": "focus-ios", // (?)
+
+    "firefox-reality": "FirefoxReality",
+
+    "firefox-reality-pc": "", // (?)
+
+    "glean-js": "", // (?)
+
+    "lockwise-android": "lockbox-android", // (?)
+
+    "lockwise-ios": "", // (?)
+
+    mach: "", // (?)
+
+    mozphab: "", // (?)
+
+    mozregression: "", // (?)
```

Comment:
I was not able to find some applications in Searchfox. Could you help point me to where I might be able to find these repos @wlach? Thanks!

Reply:
Right. Someone needs to explicitly add a repo to be indexed to searchfox. I think we can just skip these repos for now, where they're not indexed -- we can always add them at a later time, and update this list. The major ones (e.g. Firefox for Android) are.